Library/Formula/dmd.rb

```diff
@@ -65,6 +57,27 @@ def install
     end
   end
 
+  # Fix up dmd.conf to point to the correct include paths.
+  def fix_dmd_conf
+    # Homebrew's etc.install method doesn't let you overwrite etc files,
+    # so edit it in place if it's there:
+    conf = etc/"dmd.conf"
+    if conf.exist?
+      inreplace conf, /^DFLAGS=.+$/, "DFLAGS=-I#{include}/d2 -L-L#{lib}"
```

Comment:
Well, if it's doing a regex replace, it should always find `/^DFLAGS=.+$/` in the config to "replace".   (Otherwise, it's right to complain!)  I don't remember it complaining on my numerous uninstall/install tests.